src/wasm-interpreter.h

```diff
@@ -2110,6 +2110,10 @@ template<typename GlobalManager, typename SubType> class ModuleInstanceBase {
       if (target.breaking()) {
         return target;
       }
+      if (target.getSingleValue().isNull()) {
+        // TODO: remove this when we fully support non-nullable types
```

Comment:
According to the [spec](https://github.com/WebAssembly/function-references/blob/master/proposals/function-references/Overview.md#functions), `call_ref` takes a nullable reference and traps on the null value, so I think this can be the final state and we don't need this TODO?

Reply:
Good point, I thought it takes a non-null reference... fixing.